compiler/q-implant/README.md

```diff
@@ -0,0 +1,25 @@
+# q-implant
+
+_q-implant_ writes quantization parameters and weights (given as .json and .npy files) to a circle model.
```

Comment:
(moving discussion in #11579 to here)

plz explain more about "writes weights (.npy)"

CC @jinevening

Reply:
(moving discussion in https://github.com/Samsung/ONE/pull/11579/files#r1338134299 to here)

Then, q-implant should provide the structure / specification of this json and npy file.

Reply:
Willl be discussed in https://github.com/Samsung/ONE/pull/11638.